### DISCUSSION TITLE
Google Analitics + Quasar

### DISCUSSION THREAD

**Discussion by murilolivorato** [quasarframework/quasar#8824] [CLI - SSR mode]
I did the Google Analitics Verification and the Tag Manager Verification .
WHat is this sessionId  in this code ?
I am trying to do this configuration -
https://quasar.dev/quasar-cli/developing-cordova-apps/managing-google-analytics

```
import ga from 'src/helpers/analytics.js'

export default ({ router }) => {
  router.afterEach((to, from) => {
    ga.logPage(to.path, to.name, sessionId)
  })
}

```

**Comment by hawkeye64** [comment]
I am not that familiar with `ga` but I suggest looking at the code for Quasar docs to see how it was done.

**Comment by digitalit** [comment]
Hi guys,

Being running Matomo to track Quasar SSR app but now a customer insisted on using GA and i could't find any clear instructions for SSR so I did a little digging.

My solution seems to work fine and this is what I did:

1. Prepare everything in GA to get the UA-XXXXXXX code
2. Install https://matteo-gabriele.gitbook.io/vue-gtag/
3. Created a boot file gtm.js

```
import VueGtag from 'vue-gtag'

export default async ({ Vue, app, router }) => {
  if (process.env.NODE_ENV === 'production') {
    Vue.use(
      VueGtag,
      {
        config: {
          id: 'UA-XXXXXXXX-X'
        }
      },
      router
    )
  }
}
```

4. Added the boot file in quasar.conf.js

```
  boot: [
      'platform',
      'auth',
      'i18n',
      {
        server: false,
        path: 'gtm'
      },
```

Everything looks fine in:
analytics.google.com
tagassistant.google.com
tagmanager.google.com
